web/src/settings_org.ts

```diff
@@ -947,8 +972,13 @@ export function set_up_dropdown_widget_for_realm_group_settings(): void {
         realm.server_supported_permission_settings.realm,
     );
 
+    const settings_using_pills_ui = new Set([
+        "can_create_public_channel_group",
+        "can_create_private_channel_group",
+        "create_multiuse_invite_group",
```

Comment:
This is a workaorund and it can be modfied to just call the code below for settings which allow only system groups when we convert all the settings to new UI.